docs/content/installation/uninstall_driver_openshift_web.md

```diff
@@ -11,12 +14,18 @@ Perform the following steps in order to uninstall the CSI driver and operator th
 
     **Operators** > **Installed Operators** > **Operator Details**.
 
-4.  Click on the **more** menu for the **ibm-block-csi** driver and select **Delete IBMBlock CSI**.
+4. If applicable, click on the **more** menu for the **host-definer** driver and select **Delete HostDefiner**.
```

Comment:
this should be done after deleting the driver

Reply:
@matancarmeli7 please confirm what @oriyarde wrote above - I put this in per your initial instructions

Reply:
yes, @oriyarde  is right, all the uninstall should be in the same order

Reply:
updated